### PR TITLE
PRO-3078: Link invite screeners pages

### PR DESCRIPTION
**Pull request by LucaDelBuonoHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRO-3078

### Change description ###
Also implements the feature toggling of the screeners questions in:
https://tools.hmcts.net/jira/browse/PRO-3724

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: app/steps/ui/newstarteligibility/template.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    </div>
+  </div>
+  <div class="form-group">
+    <p><a href="/new-will-left" class="button button-start">{{common.confirm}}</a></p>
+  </div>
+
+  <p><a href="/tasklist">{{content.paragraph4}}</a></p>
+</main>
+
+{% endblock %}
```
Comment (shepda) [inline app/steps/ui/newstarteligibility/template.html]: We're not displaying the 'Do you need help?' link on this page, is that correct?

File: app/steps/ui/will/newleft/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const ValidationStep = require('app/core/steps/ValidationStep'),
+    json = require('app/resources/en/translation/will/left.json');
+module.exports = class NewWillLeft extends ValidationStep {
```
Comment (shepda) [inline app/steps/ui/will/newleft/index.js]: Please export the module at the bottom of the file, see https://github.com/hmcts/probate-tech-docs/blob/master/javascript-coding-style.md#module-exports for an example

File: app/steps/ui/will/newleft/index.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+const ValidationStep = require('app/core/steps/ValidationStep'),
+    json = require('app/resources/en/translation/will/left.json');
```
Comment (shepda) [inline app/steps/ui/will/newleft/index.js]: Please put each variable declaration on a new line, see https://github.com/hmcts/probate-tech-docs/blob/master/javascript-coding-style.md#variable-declarations for an example

File: app/steps/ui/will/newleft/index.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+    nextStepOptions() {
+        const nextStepOptions = {
+            options: [
+                {key: 'left', value: json.optionYes, choice: 'withWill'}
+            ]
+        };
+        return nextStepOptions;
+    }
+};
```
Comment (shepda) [inline app/steps/ui/will/newleft/index.js]: Please put 'use strict'; on the first line of this file, see https://github.com/hmcts/probate-tech-docs/blob/master/javascript-coding-style.md#strict-mode for an example

File: app/steps/ui/will/neworiginal/index.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+const ValidationStep = require('app/core/steps/ValidationStep'),
+    json = require('app/resources/en/translation/will/original.json');
```
Comment (shepda) [inline app/steps/ui/will/neworiginal/index.js]: Please put each variable declaration on a new line, see https://github.com/hmcts/probate-tech-docs/blob/master/javascript-coding-style.md#variable-declarations for an example

File: app/steps/ui/will/neworiginal/index.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+module.exports = class NewWillOriginal extends ValidationStep {
```
Comment (shepda) [inline app/steps/ui/will/neworiginal/index.js]: Please export the module at the bottom of the file, see https://github.com/hmcts/probate-tech-docs/blob/master/javascript-coding-style.md#module-exports for an example

File: app/steps/ui/will/neworiginal/index.js [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+    nextStepOptions() {
+        const nextStepOptions = {
+            options: [
+                {key: 'original', value: json.optionYes, choice: 'isOriginal'}
+            ]
+        };
+        return nextStepOptions;
+    }
+};
```
Comment (shepda) [inline app/steps/ui/will/neworiginal/index.js]: Please put 'use strict'; on the first line of this file, see https://github.com/hmcts/probate-tech-docs/blob/master/javascript-coding-style.md#strict-mode for an example

**Comment by bmj-vsalivendra** [comment]
This PR doesn't have [PREVIEW] prefix hence I'm closing and will be create another one with [PREVIEW].

**Pull request closed without merging**